.github/workflows/create-driver-channel.yml

```diff
@@ -0,0 +1,101 @@
+name: Create Driver Channel
+on:
+  pull_request:
+    types: [opened, synchronize]
```

Comment:
What will happen if additional changes are added to the PR after it's opened? Will those be pushed to the channel?

Reply:
`synchronize` = updates to PR like new commits

Reply:
So a new channel would be created each time, right? Do we have a way to "deactivate" the previous invites that only the latest one can be used?

Reply:
The first functioning version would delete the previous comment before adding a new one so only one invite _link_ will be visible at the same time, but the invites will all point to the same channel anyway that will continue to get its driver updated, so which invite link you use should not matter.